active/NavigationView/BottomItems/BottomMenuItems.md

```diff
@@ -0,0 +1,151 @@
+
+
+NavigationView provides a built-in [Settings item](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.settingsitem?view=winui-2.3), which is always positioned at the bottom or right of the Pane, and a [PaneFooter](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.panefooter?view=winui-2.3) area that positions content just before (either above or to the left) of the Settings item.
+Although an app developer can place any content into PaneFooter, including other NavigationViewItems, this content will not participate in the selection model and animations of the primary navigation list and Settings item.
+
+_Open question:_ Should PaneFooter be deprecated because this new API is a direct replacement?
+
+Some apps may choose to visually group navigation items into top-aligned and bottom-aligned lists.
+While this grouping is possible today using navigation view's [PaneFooter](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.panefooter?view=winui-2.3) property, the approach has interaction downsides that this feature will correct.
+
+![NavigationView in Left display mode showing FooterMenuItems](FooterItems_Left.png)
+
+![NavigationView in Top display mode showing FooterMenuItems](FooterItems_Top.png)
+
+By adding FooterMenuItems, we'll unify the [Settings item](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.settingsitem?view=winui-2.3) with other navigation items in its immediate proximity. The menu items that appear at the top or left of the NavigationView will be a part of their own list, and the bottom or right menu items will be a part of their own list, including the Settings item. 
+
+As a result, keyboard and assistive technology users will feel that the footer list behavior meets their expectations -  the top and bottom menu items will be specified as two separate lists, but it will be smooth to navigate from one to the other. See more in Accessibility section below.
+Selection behavior and animations will also feel better integrated into the whole NavigationView control, as bottom menu items will participate in the same selection model as top menu items. 
+
+# Description 
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+FooterMenuItems is a list of navigation items similar to the MenuItems list.
+FooterMenuItems allows apps to visually group their navigation items into top-aligned and bottom-aligned or left-aligned and right-aligned groups. 
+The default Settings item will be included as the last item in the FooterMenuItems collection. As a result, adding items into FooterMenuItems will by default display them before the Settings item. The Settings item can still be toggled using the `IsSettingsVisible` property.
```

Comment:
That's a good point, thanks for bringing it up. I think the original idea was to have the Settings item be a part of the FooterMenu collection. However, this would force us to change the logic behind how we toggle the Settings item's visibility (it would no longer be detached). 

You're correct that today Settings item is detached - it shows up in SelectedItem but not MenuItems, which is why we'd have to change things up with toggling its visibility as I just mentioned.

It would not be ideal for Settings to be able to be removed from the FooterMenu collection, but I'm not sure the best place to put it if we want to keep the grouping of settings + other footer menu items in terms of them being and behaving as one list.

@ojhad and @MikeHillberg let me know if you have opinions on this dilemma.

Reply:
Making the decision to include the settings item as a member of the FooterMenu collection. It makes more sense for the overall design of the control, and although one will be able to remove it manually from the collection, it doesn't break anything or change any other functionality.